Common/QuantConnect.csproj

```diff
@@ -274,7 +274,9 @@
     <Compile Include="Data\Custom\Estimize\EstimizeEstimate.cs" />
     <Compile Include="Data\Custom\Estimize\EstimizeRelease.cs" />
     <Compile Include="Data\Custom\PsychSignal\PsychSignalSentiment.cs" />
-    <Compile Include="Data\Custom\USEnergyInformation.cs" />
+    <Compile Include="Data\Custom\USEnergy\USEnergy.cs" />
+    <Compile Include="Data\Custom\USEnergy\USEnergyAPI.cs" />
+    <Compile Include="Data\Custom\USEnergy\USEnergyCategory.cs" />
```

Comment:
I think to prevent an explosion of key-classes with unique names we should keep a convention or better include the keys in the class itself. i.e. Should we mode all the country statics -> USEnergy.

This would move it more inline with Trading Economics https://github.com/QuantConnect/Lean/blob/master/Common/Data/Custom/TradingEconomics/TradingEconomics.cs